server/helpers/passport.ts

```diff
@@ -0,0 +1,22 @@
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+require('dotenv').config();
```

Comment:
shouldn't need this line since we've already called it earlier in the server

Reply:
Because this one is currently not in use, we could get rid of this file entirely if we decide to keep the passport function where it is in the server/index.ts. Removing line 2 from here stops the compiler.